### ISSUE TITLE
AAX files can not be played

### ISSUE THREAD

**Issue by pichfl** [GianniCarlo/Audiobook-Player#110]
Trying to import a file from Audible has unexpected effects. The file correctly shows the album artwork and meta info but can not be played. If the import happens while another, valid file is playing, its content will be played instead of the imported file. Restarting the player and trying to play the AAX yields in a player screen that stays paused.

We should add some handling for this scenario, most likely limiting the allowed file formats to the tested suspects as it is unlikely we would be able to get clearing from Amazon to decode the files.

**Comment by GianniCarlo** [comment]
👌

**Comment by GianniCarlo** [comment]
From what I read from this answer: https://apple.stackexchange.com/a/243670 processing AAX files is out of scope for this project and we should exclude that file from the accepted ones.

Excerpt:
> aax files are encrypted mp4 / m4a / m4b files so the ffmpeg command does not re-encode audio and it preserves metadata such as chapters

Then it goes on about using the user/pswd of the account to get a token and decrypt the AAX file

**Comment by pichfl** [comment]
Yeah, we should not try to be too clever here. Excluding the format is definitely the best solution for us right now.

**Comment by GianniCarlo** [comment]
I thought about putting an `if file.ext == .aax` in the processing file flow, but it was too messy. Instead I opted on fixing/streamlining the error handling when a file fails to load, and show an alert that it might be due to DRM.

That way it's up to the user if he wants to delete that unusable file or not 🤷‍♂️

**Comment by Wiredframe** [comment]
Is there a chance for (unencrypted) m4a support in a future update?
